packages/components/src/components/file-dropzone/file-dropzone.tsx

```diff
@@ -89,6 +89,20 @@ const PreviewSingle = (props: TPreviewSingle) => {
 };
 
 const FileDropzone = ({ className, noClick = false, ...props }: TFileDropzone) => {
+    const { validation_error_message, message } = props;
+
+    const RenderErrorMessage = React.useCallback(() => {
+        if (typeof message === 'function') return <>{message(open)}</>;
```

Comment:
```suggestion
        if (typeof message === 'function') <>{message(open)}</>;
```

Dose it work like this?

Reply:
@niloofar-deriv it can't work that way because there's another return component in the same function so it will give error if i don't declare return on the condition